src/build.gradle.kts

```diff
@@ -0,0 +1,251 @@
+import org.apache.tools.ant.filters.ReplaceTokens
+import java.util.*
+
+plugins {
+    java
+    checkstyle
+    `maven-publish`
+    id("org.springframework.boot") version "3.4.4"
+    id("io.spring.dependency-management") version "1.0.11.RELEASE"
+    id("org.owasp.dependencycheck") version "12.1.1"
+    jacoco
+}
+
+checkstyle {
+    toolVersion = "10.23.1"
+    group = "verification"
+    configFile = file("${project.projectDir}/config/checkstyle/checkstyle-config.xml")
+}
+
+dependencyCheck {
+//    cveValidForHours = 12
+//    assemblyAnalyzerEnabled = false
+//    enableExperimental = false
+    //TODO Do we need an apiKey? Otherwise: One or more exceptions occurred during analysis:
+    //	UpdateException: Error updating the NVD Data; the NVD returned a 403 or 404 error
+//    nvd {
+//        apiKey=
+//    }
+    outputDirectory = "${project.projectDir}/build/reports/dependency-check-report"
+    suppressionFile = "${project.projectDir}/src/dependency-check-suppressions.xml"
+    formats = listOf("HTML", "XML")
+}
+
+repositories {
+    mavenLocal()
+    maven {
+        url = uri("https://artifactory.niis.org/xroad-maven-releases")
+        mavenContent {
+            releasesOnly()
+        }
+    }
+    maven {
+        url = uri("https://artifactory.niis.org/xroad-maven-snapshots")
+        mavenContent {
+            snapshotsOnly()
+        }
+    }
+    maven {
+        url = uri("https://repo.maven.apache.org/maven2/")
+    }
+    mavenCentral()
+}
+
+ext {
+    set("xrd4j.version", "0.6.0")
+    set("java.version", "21")
+    set("jdk.version", "21")
+    set("project.build.sourceEncoding", "UTF-8")
+    set("project.build.resourceEncoding", "UTF-8")
+    set("project.reporting.outputEncoding", "UTF-8")
+    set("app.home", "/var/lib/tomcat/webapps")
+    set("server.port", "9898")
+}
+
+dependencies {
+    // SpringBoot
+    implementation("org.springframework.boot:spring-boot-starter-web:3.4.4")
+    implementation("org.springframework.boot:spring-boot-starter-aop:3.4.4")
+    implementation("org.springframework.boot:spring-boot-starter-tomcat:3.4.4")
+    testImplementation("org.springframework.boot:spring-boot-starter-test:3.4.4") {
+        exclude(group = "com.vaadin.external.google", module = "android-json")
+    }
+
+    // xrd4j
+    implementation("org.niis.xrd4j:common:0.6.0")
+    implementation("org.niis.xrd4j:client:0.6.0")
+    implementation("org.niis.xrd4j:server:0.6.0")
+    implementation("org.niis.xrd4j:rest:0.6.0")
+
+    // Lombok
+    compileOnly("org.projectlombok:lombok:1.18.30")
+    annotationProcessor("org.projectlombok:lombok:1.18.30")
+    testImplementation("org.projectlombok:lombok:1.18.30")
+    testAnnotationProcessor("org.projectlombok:lombok:1.18.30")
```

Comment:
We should decide here if we use lombok everywhere or nowhere, currently it is not used in the main code anymore but is still used in tests.

I would either fix it so that we can use it in the main code as well, or drop it all together. Either option is ok by me, but lets not keep it this way.